src/plugins/irc-events/kick.js

```diff
@@ -5,7 +5,7 @@ module.exports = function(irc, network) {
 	var client = this;
 	irc.on("kick", function(data) {
 		var from = data.nick;
-		var chan = _.findWhere(network.channels, {name: data.channel});
+		var chan = _.find(network.channels, {name: data.channel});
```

Comment:
I'm sure there is an ES5 equivalent here too, but I won't be as picky a with `.map()` and `.filter()` which are now well accepted.